charts/trow/templates/webhooks/tls-certmanager.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if and (include "webhook.enabled" .) .Values.webhooks.tls.certmanager.enabled }}
+{{- include "webhook.validateTlsGenValues" . -}}
+# Create a selfsigned Issuer, in order to create a root CA certificate for
+# signing webhook serving certificates
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: {{ include "trow.fullname" . }}-cm-admission-self-signed-issuer
+  namespace: {{ .Release.Namespace }}
+spec:
+  selfSigned: {}
+---
+# Generate a CA Certificate used to sign certificates for the webhook
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ include "trow.fullname" . }}-cm-admission-root-cert
+  namespace: {{ .Release.Namespace }}
+spec:
+  secretName: {{ include "trow.fullname" . }}-cm-admission-root-cert
+  duration: {{ .Values.webhooks.tls.certmanager.rootCert.duration | default "43800h0m0s" | quote }}
+  issuerRef:
+    name: {{ include "trow.fullname" . }}-cm-admission-self-signed-issuer
+  commonName: "ca.webhook.trow"
+  isCA: true
+  subject:
+    organizations:
+      - trow-registry
+---
+# Create an Issuer that uses the above generated CA certificate to issue certs
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: {{ include "trow.fullname" . }}-cm-admission-root-issuer
+  namespace: {{ .Release.Namespace }}
+spec:
+  ca:
+    secretName: {{ include "trow.fullname" . }}-cm-admission-root-cert
+---
+# generate a server certificate for the apiservices to use
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ include "trow.fullname" . }}-admission
```

Comment:
@awoimbee Do you want to use `-webhooks` as the suffix, to match what is used in the annotations in the *webhook.yaml files?